VimR/VimR/PrefMiddleware.swift

```diff
@@ -20,6 +20,7 @@ final class PrefMiddleware: MiddlewareType {
     do {
       let dictionary: [String: Any] = try dictEncoder.encode(appState)
       defaults.set(dictionary, forKey: PrefMiddleware.compatibleVersion)
+      defaults.synchronize()
```

Comment:
Do we need this? According to the [doc](https://developer.apple.com/documentation/foundation/userdefaults/1414005-synchronize), it's not needed.

Reply:
I had a really hard time getting prefs to stick without it and found references to it being needed elsewhere.  It's possible that's because I was often killing and restarting vimR in the debugger, but I think it doesn't sync quickly by default.